lighthouse-core/gather/driver/navigation.js

```diff
@@ -94,9 +96,14 @@ async function gotoURL(driver, url, options) {
   await session.sendCommand('Page.enable');
   await session.sendCommand('Page.setLifecycleEventsEnabled', {enabled: true});
 
-  // No timeout needed for Page.navigate. See https://github.com/GoogleChrome/lighthouse/pull/6413
-  session.setNextProtocolTimeout(Infinity);
-  const waitforPageNavigateCmd = session.sendCommand('Page.navigate', {url});
+  let waitForPageNavigateCmd;
```

Comment:
`waitForPageNavigateCmd` -> `waitForPageNavigatePromise` now?

Reply:
actually, both of those kinda conflict with the `waitForNavigated` down below... idea for better name?

Reply:
`waitForNavigateTriggered`?